jsonrpc-types/Cargo.toml

```diff
@@ -15,12 +15,13 @@ serde_json = "1.0"
 serde_derive = "1.0"
 rustc-serialize = "0.3"
 uuid = { version = "0.7", features = ["v4"] }
+cita-crypto = { path = "../cita-crypto", optional = true }
```

Comment:
Not perfect, the `optional` is useless.

Reply:
Yea. But there's no way around it. Let's refactor code later.

Reply:
I mean, if you remove this `optional = true`, this PR still can be passed.
At least, one feature have to be selected, so this crate is not `optional`, it is **necessary**.